### PR TITLE
Ameerul /P2PS 1082 Refactor Block User Modal

### PR DESCRIPTION
**Pull request by ameerul-deriv**
## Changes:

Please provide a summary of the change.
- Refactored Block User Modal component
- Added test cases

### Screenshots:

Please provide some screenshots of the change.
Screenshot 2023-07-10 at 3 18 56 PM

### CODE REVIEW

File: packages/p2p/src/components/modal-manager/modals/block-user-modal/__tests__/block-user-modal.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { useStores } from 'Stores/index';
```
Comment (farhan-nurzi-deriv) [inline packages/p2p/src/components/modal-manager/modals/block-user-modal/__tests__/block-user-modal.spec.tsx]: **non-blocking**
```suggestion
import { useStores } from 'Stores';
```

**Review by farhan-nurzi-deriv — APPROVED** [review]

File: packages/p2p/src/components/modal-manager/modals/block-user-modal/__tests__/block-user-modal.spec.tsx (hunk 1 above)
Comment (ameerul-deriv) [reply packages/p2p/src/components/modal-manager/modals/block-user-modal/__tests__/block-user-modal.spec.tsx]: man eslint was complaining hahah but let me see what I can do

Comment (farhan-nurzi-deriv) [reply packages/p2p/src/components/modal-manager/modals/block-user-modal/__tests__/block-user-modal.spec.tsx]: Maybe ignore it first, there is fix in my order pages refactoring PR 😆

**Review by niloofar-deriv — APPROVED** [review]
LGTM :+1:

**Pull request merged**